app/src/main/java/com/tvtracker/dao/IMediaItemDAO.kt

```diff
@@ -14,4 +14,6 @@ interface IMediaItemDAO {
     @GET("/")
     fun searchByImdbId(@Query("i") imdbId: String): Call<MediaItem>
 
+    @GET("/")
+    fun addWatchedMovie(userId: Int, movieId: Int)
```

Comment:
A few issues here:
- Does the endpoint support this?  If not, this is dead/non working code, which is technical debt.
- "Add" sounds like it should be a POST or PUT, not a GET.